tests/src/main/java/com/hazelcast/simulator/tests/map/IntByteMapTest.java

```diff
@@ -77,11 +79,17 @@ public void tearDown() throws Exception {
 
     @Warmup(global = false)
     public void warmup() throws InterruptedException {
-        MapStreamer<Integer, Object> streamer = MapStreamerFactory.getInstance(map);
         Random random = new Random();
+        values = new byte[valueCount][];
+        for (int k = 0; k < values.length; k++) {
+            int delta = maxSize - minSize;
+            int length = delta == 0 ? minSize : minSize + random.nextInt(maxSize - minSize);
+            values[k] = generateByteArray(random, length);
+        }
+
+        MapStreamer<Integer, Object> streamer = MapStreamerFactory.getInstance(map);
         for (int key : keys) {
-            Object value = generateByteArray(random, valueSize);
-            streamer.pushEntry(key, value);
+            streamer.pushEntry(key, values[random.nextInt(values.length)]);
```

Comment:
then you still have have a pattern since you repeat the same pattern for every 'valueCount' entries.